lms/static/styles/_typography.scss

```diff
@@ -1,3 +1,6 @@
+@use "variables" as *;
```

Comment:
In every other module, this is namespaced. Is there a reason it's still wildcarded here?

Reply:
@lyzadanger 

missed this one, also missed mixins.sass which is now updated too